Design.md

```diff
@@ -309,3 +332,34 @@ Get the platform usage of global (without query parameter), tenant (with tenant_
         * `key` : `offset` and `value` : `marker`
     1. `AssumeRoleCache` for all the `User` and `S3 Credential` APIs
         * `key` : `RoleArn` and `value` : `Credentials`
+
+## SecretKey Encryption Strategy
+
+1. Secret keys are encrypted and decrypted using a cipher algorithm and secret password provided in the `crypto.yml` config file.
+1. A `key-id` property will be provided for each `secret-password` in the `crypto.yml` config file to maintain the version of the key and will be used in the Key rotation.
```

Comment:
It's not a password. Let's not call it `password`. Also, when I read this not knowing what comes after, I'd assume this says there is some `secret-password` key in some YAML structure, which is not the case. TBH I'm not sure what this sentence conveys.

Reply:
Fixed in 1779625